1주차/4796/4796_python_제로.py

```diff
@@ -0,0 +1,16 @@
+case = 1
+
+while 1:
+    L, P, V = map(int, input().split())
+    if (L == 0 and P == 0 and V==0):
+        break
+
+    step1 = V // P
+    step2 = V - step1 * P
+    if (L < step2):
+        step2 = L
```

Comment:
`min` 메서드를 사용해서 간단하게 줄여볼 수 있겠네요
```suggestion
    step2 = min(L, step2)
```

Reply:
굳